tests/envs/compiler_env_test.py

```diff
@@ -47,5 +51,56 @@ def test_state_from_csv_invalid_format():
     assert str(ctx.value).startswith("Failed to parse input: `abcdef`: ")
 
 
+@pytest.fixture(scope="function")
+def env() -> CompilerEnv:
```

Comment:
This doesn't seem to be called?

Reply:
This declares a [pytest fixture](https://docs.pytest.org/en/stable/fixture.html#back-to-fixtures) so that tests in this file that take an `env` argument will use this.